### PR TITLE
Recurring Donation Label Refactor into Apex Constants

### PR DESCRIPTION
**Pull request by force2b**
- Remove all uses of legacy Recurring Donation custom labels with hardcoded Apex Constants. This addresses an issue with labels that are not supposed to be translated in our underlying packages, but due to a packaging limitation the translations still exist in the package even though they were remove two years ago.

# Critical Changes

# Changes

- Deprecate legacy custom labels that were previously used by a much older version of Recurring Donations. These labels have not been needed since Salesforce introduced Picklist Api Values in the Spring '17 release, however they have still exist in the package and could cause issues if the values were translated.

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
** lurch: attach W-039017

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HrLIQA0" target="_blank">W-039017</a>

File: src/classes/RD_Constants.cls [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
```
Comment (RadSFDeveloper) [inline src/classes/RD_Constants.cls]: The year needs update here and line 2.

File: src/classes/RD_Constants.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>47.0</apiVersion>
```
Comment (RadSFDeveloper) [inline src/classes/RD_Constants.cls-meta.xml]: Version 46.0?

**Comment by RadSFDeveloper** [comment]
@sam-knox , can you please verify Changes section?

File: src/classes/RD_Constants.cls-meta.xml (hunk 2 above)
Comment (force2b) [reply src/classes/RD_Constants.cls-meta.xml]: Since v48 is going live next week, we're fine with using v47 on new classes.

**Comment by heidmary** [comment]
@force2b The WI ACs are all tested and pass. Just noticing that the failing Feature Test - Robot errors are all language related. Are they expected? Is it okay for me to merge this PR with those tests failing? Thanks.

**Comment by heidmary** [comment]
Rebuild fixed the language related robot failures. Remaining failures are known. Merging.

**Pull request merged**